cmd/system/powershell.go

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) arkade author(s) 2024. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package system
+
+import (
+	"fmt"
+	"os"
+	"path"
+	"strings"
+
+	"github.com/alexellis/arkade/pkg/archive"
+	"github.com/alexellis/arkade/pkg/env"
+	"github.com/alexellis/arkade/pkg/get"
+	"github.com/spf13/cobra"
+)
+
+func MakeInstallPowershell() *cobra.Command {
+	command := &cobra.Command{
+		Use:   "pwsh",
+		Short: "Install Powershell",
+		Long:  `Install Powershell cross-platform task automation solution.`,
+		Example: `  arkade system install pwsh
+  arkade system install pwsh --version <version>`,
+		SilenceUsage: true,
+	}
+
+	command.Flags().StringP("version", "v", "", "The version or leave blank to determine the latest available version")
+	command.Flags().String("path", "/usr/local/bin", "Installation path, where a powershell subfolder will be created")
```

Comment:
Hi, I know most of what we install goes here, but this isn't the recommended path for PowerShell

Could you consult the docs and update?

Reply:
Done